Source/engine/load_file.hpp

```diff
@@ -132,20 +132,21 @@ struct MultiFileLoader {
 		StaticVector<SFile, MaxFiles> files;
 		StaticVector<uint32_t, MaxFiles> sizes;
 		size_t totalSize = 0;
-		for (size_t i = 0; i < numFiles; ++i) {
+		for (size_t i = 0, j = 0; i < numFiles; ++i) {

```

Comment:
I'm failing to see where j is being defined?

Reply:
Right here, you can define any number of variables in a for loop.

Reply:
oh wait it's a `,` :D